README.md

```diff
@@ -1,5 +1,7 @@
 # CDNJS - the best front-end resource CDN for free!
 
+[![Greenkeeper badge](https://badges.greenkeeper.io/cdnjs/cdnjs.svg)](https://greenkeeper.io/)
```

Comment:
@PeterDaveHello The `svg` shows greenkeeper is disable. Would this be ok?

Reply:
Ummm.. 
I think the link be modified to https://badges.greenkeeper.io/greenkeeperio/badges.svg is good.
So we are going to have a new bot(greenkeeper)?
Interesting~ 😃